R/RunModule.R

```diff
@@ -95,15 +87,16 @@ runModule <- function(analysisSpecifications, keyringSettings, moduleIndex, exec
         renv::use(lockfile = "renv.lock")
       }
 
-      # NOTE injected variable isResultsExecution - will look strange outside of Strategus definition
+      # NOTE: injected variable isResultsExecution - will look strange outside of Strategus definition
+      # NOTE: retrieveConnectionDetails function is injected by withModuleRenv
       if (isCdmExecution) {
-        connectionDetails <- Strategus::retrieveConnectionDetails(
```

Comment:
Will keyring and ParallelLogger still have issues? They are called in other places in this script and that would be harder to fix? I've been trying to break this and it seems ok though

Reply:
That is still possible - the reason why these work now is because they are included in the module renv.lock environment. Assuming the module is restored properly AND has these dependencies it will work. I'd like to formalize these type of dependencies as mentioned in #69.